build-tools/installers/create-installers.targets

```diff
@@ -0,0 +1,113 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="..\..\Configuration.props" />
+  <Import Project="..\scripts\XAVersionInfo.targets" />
+  <UsingTask AssemblyFile="..\..\bin\Build$(Configuration)\xa-prep-tasks.dll"   TaskName="Xamarin.Android.BuildTools.PrepTasks.ReplaceFileContents" />
+  <PropertyGroup>
+    <RootBuildDir>..\..\bin\$(Configuration)\</RootBuildDir>
```

Comment:
I half wonder if this should contain an `$(MSBuildThisFileDirectory)` or a `Path.GetFullPath()` invocation to increase resiliency...